### ISSUE TITLE
Personal notification stream with filter for notifications I am invested in

### ISSUE THREAD

**Issue by holljen** [integrations/microsoft-teams#36]
I have a manager role in my company and I do not have time to follow all PR's, issues etc. However, I do spend time on the ones I consider important and I would like to follow these. 

Therefore it would be nice if it was possible to have a personal notification stream where in addition to the current subscription options it is possible to filter on issues, tickets etc where I am invested in some way. For example:
- created the issue, PR etc.
- am a reviewer
- have replied
- have set a thumbs up
- probably more...

**Comment by ashokirla** [comment]
Yes, we have plans to expand our personal app scope. We will definitely consider your inputs.

**Comment by ashokirla** [comment]
@holljen Hope you are doing good. We are currently working on GA features and we would like to sync up with you and get your feedback. Can you please let us know if it is possible to have a quick call.

**Comment by holljen** [comment]
Sorry for the late reply, it was a busy week. I am free for a call whenever i am not sleeping CET 😀

**Comment by jeremypoulter** [comment]
Just tagging on my 2p worth here as I have a similar requirement: just spent the last couple of hours tidying up my GitHub notifications so from my POV what would be ideal is to expose the GitHub notification feed in Teams, as in the Bell that is in the top/right of the GitHub page
![image](https://user-images.githubusercontent.com/6116992/105045102-6bafac00-5a5f-11eb-9b8a-ed2c9b1a604f.png)

**Comment by Atasnef2022** [comment]
I hope you take the necessary action. We have warehouses around the world: Our enterprise is restricted and controlled by government agencies and organizations from Saudi Arabia to East Asia to Europe to America. He can't interfere. Our project: qualifying and classifying professional institutions and individuals and including them in the integration network. According to the project code in full with the terms and specifications of the service provider
